skopt/tests/test_common.py

```diff
@@ -418,11 +418,11 @@ def test_consistent_x_iter_dimensions(minimizer):
 @pytest.mark.parametrize("minimizer",
                          [gp_minimize, forest_minimize, gbrt_minimize])
 def test_early_stopping_delta_x(minimizer):
-    n_calls = 10
+    n_calls = 11
     res = minimizer(bench1,
                     callback=DeltaXStopper(0.1),
                     dimensions=[(-1., 1.)],
-                    x0=[[-0.1], [0.9], [0.1]],
+                    x0=[[-0.1], [0.1], [-0.9]],
```

Comment:
Had to update this test after changing the seeding procedure as it relies on a particular sequence of numbers to trigger the stopping condition.